src/unix/bsd/apple/mod.rs

```diff
@@ -4687,12 +4687,13 @@ pub const MNT_SNAPSHOT: ::c_int = 0x40000000;
 pub const MNT_NOBLOCK: ::c_int = 0x00020000;
 
 // sys/spawn.h:
-pub const POSIX_SPAWN_RESETIDS: ::c_int = 0x01;
-pub const POSIX_SPAWN_SETPGROUP: ::c_int = 0x02;
-pub const POSIX_SPAWN_SETSIGDEF: ::c_int = 0x04;
-pub const POSIX_SPAWN_SETSIGMASK: ::c_int = 0x08;
-pub const POSIX_SPAWN_SETEXEC: ::c_int = 0x40;
-pub const POSIX_SPAWN_START_SUSPENDED: ::c_int = 0x80;
+pub const POSIX_SPAWN_RESETIDS: ::c_int = 0x0001;
```

Comment:
They are declared with 4 digits in `sys/spawn.h`, and having them all 4 digits aligns a bit better so I made them all 4 digits. I can revert that if you prefer.
`posix_spawnattr_setflags` and `posix_spawnattr_getflags` use `short` so I think they should be declared `::c_short` (in C they are defined with `#define` so not really typed), but that would probably break compatibility so I left the type as-is, even for the `_POSIX_SPAWN_DISABLE_ASLR` I added.